.github/workflows/deploy.yml

```diff
@@ -1,4 +1,4 @@
-name: Logger deploy build
+name: Deploy
```

Comment:
A tuto action este vobec potrebujeme na nieco?

Reply:
Nechal som ju tam lebo teoreticky keď sa nemení Dockerfile tak ten image na ktorom sa to testuje môže byť ten z Dockerhubu. Všetky súbory z repa by sa mali do kontajnera prekopírovať(?)